modules/nvidia-gpu-aws-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
```

Comment:
This module is good but it's not for GPU-enablement when you install you cluster.  It's more for GPU-enabled node after you've already installed your cluster. 

What you need to do is add it to the end of the assembly on creating Machines/MachineSets for AWS: https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-aws.html

As an example, Google Cloud Platform already has its post-install Machine/MachineSet config in the docs: https://docs.openshift.com/container-platform/4.11/machine_management/creating_machinesets/creating-machineset-gcp.html#machineset-gcp-enabling-gpu-support_creating-machineset-gcp

We just need to do the same thing for the AWS instructions.

Reply:
Moved **nvidia-gpu-aws-adding-a-gpu-node.adoc** to the **creating-machineset-aws.adoc** assembly.